src/main/kotlin/no/nav/familie/tilbake/api/e2e/AutotestController.kt

```diff
@@ -39,4 +49,15 @@ class AutotestController(private val taskRepository: TaskRepository) {
                                  }))
         return Ressurs.success("OK")
     }
+
+    @PutMapping(path = ["/behandling/{behandlingId}/endre/saksbehandler/"],
+                produces = [MediaType.APPLICATION_JSON_VALUE])
+    @Rolletilgangssjekk(minimumBehandlerrolle = Behandlerrolle.SYSTEM,
+                        handling = "endre ansvarlig saksbehandler")
+    fun endreAnsvarligSaksbehandler(@PathVariable("behandlingId") behandlingId: UUID,
```

Comment:
"behnadlingId" i @PathVariable("behandlingId") er unødvendig så lenge parameteren har samme navn.

Reply:
Fjernet "behandlingId", tilsvarende for ansvarligsaksbehandler som ble gjort om til PathVariable